packages/api/endpoints/async-operations.js

```diff
@@ -147,9 +124,11 @@ async function post(req, res) {
     let apiDbRecord;
     await createRejectableTransaction(knex, async (trx) => {
       const pgRecord = await asyncOperationPgModel.create(trx, dbRecord, ['*']);
-      apiDbRecord = await translatePostgresAsyncOperationToApiAsyncOperation(pgRecord[0]);
-      await indexAsyncOperation(esClient, apiDbRecord, process.env.ES_INDEX);
+      apiDbRecord = translatePostgresAsyncOperationToApiAsyncOperation(pgRecord[0]);
     });
```

Comment:
If creation record fails, exception will throw, and `apiDbRecord === undefined` seems redundant.

Reply:
@jennyhliu 😓  correct you. are - this was an issue with ts-check asserting undefined as possible.   Removed: 

5b10ae57861